.github/workflows/cover.yaml

```diff
@@ -23,10 +23,7 @@ jobs:
       with:
         go-version: '1.20'
     - run: "PATH=/usr/local/go/bin:$PATH make test-cover"
-    - uses: Wandalen/wretry.action@a163f62ae554a8f3cbe27b23db15b60c0ae2e93c # v1.3.0 Retry codecov upload. It is flaky due to a known issue https://community.codecov.com/t/upload-issues-unable-to-locate-build-via-github-actions-api/3954
```

Comment:
It does not appear that this retry wrapping is actually working. And perhaps it never did?

- https://github.com/Wandalen/wretry.action/issues/75